.travis.yml

```diff
@@ -37,36 +35,39 @@ jobs:
         - echo "travis_fold:start:build\n$(tput bold)Build QField $(tput sgr0)"
         - ./scripts/ci/pull_translations.sh
         - docker-compose -f .docker/testing/docker-compose-travis.yml run qgis /usr/src/.docker/testing/build-test.sh
-      if: type = pull_request OR tag IS present OR branch = master
+
+    - stage: test
```

Comment:
We are already now quite often short on parallel travis jobs, is it required to add one more?

Reply:
ok, I can drop it, although it lasts 40 sec

Reply:
done